.github/workflows/db-charm-tests.yaml

```diff
@@ -11,20 +11,32 @@ jobs:
         charm-repo:
           - "canonical/postgresql-operator"
           - "canonical/postgresql-k8s-operator"
-# TODO: uncomment once we've fixed https://github.com/canonical/operator/issues/987
-#          - "canonical/mysql-operator"
-#          - "canonical/mysql-k8s-operator"
+          - "canonical/mysql-operator"
+          - "canonical/mysql-k8s-operator"
 
     steps:
       - name: Checkout the ${{ matrix.charm-repo }} repository
         uses: actions/checkout@v3
         with:
           repository: ${{ matrix.charm-repo }}
 
+      - name: Checkout the operator repository
+        uses: actions/checkout@v3
+        with:
+          path: myops
+
+      - name: Install patch dependencies
+        run: pip install poetry==1.6.1
```

Comment:
Can we broaden this to any 1.x version, I think the syntax is `poetry~=1.6` (but double-check that).

Reply:
Is there a reason we pin tox to specifically 4.2.8? If not, could we broaden both? (I know there's no 4.2 beyond 4.2.8 and it's unlikely that there ever will be, but it would set a pattern for when we eventually upgrade tox).

Reply:
Yeah, let's broaden both. In other tests we just `pip install tox`, which is probably okay for a CLI tool, though arguably `~=4.2` would be best.